doc/users/explain/fonts.rst

```diff
@@ -119,14 +109,90 @@ This is especially helpful to generate *really lightweight* documents.::
     fig.savefig("AFM_PDF.pdf", format="pdf")
     fig.savefig("AFM_PS.ps", format="ps)
 
-.. note::
-  These core fonts are limited to PDF and PS backends only; they can not be
-  rendered in other backends.
 
-  Another downside to this is that while the font metrics are standardized,
-  different PDF viewer applications will have different fonts to render these
-  metrics. In other words, the **output might look different on different
-  viewers**, as well as (let's say) Windows and Linux, if Linux tools included
-  free versions of the proprietary fonts.
+Fonts in SVG
+------------
+
+Text can output to SVG in two ways controlled by the :rc:`svg.fonttype`
+rcparam:
+
+- as a path (``'path'``) in the SVG
+- as string in the SVG with font styling on the element (``'none'``)
+
+
+When saving via ``'path'`` Matplotlib will compute the path of the glyphs used
+as vector paths and write those to the output.  The advantage of this is that
+the SVG will look the same on all computers independent of what fonts are
+installed.  However the text will not be editable after the fact.
+In contrast saving with ``'none'`` will result in smaller files and the
+text will appear directly in the markup.  However, the appearance may vary
+based on the SVG viewer and what fonts are available.
+
+Fonts in Agg
+------------
 
-  This also violates the *what-you-see-is-what-you-get* feature of Matplotlib.
+To output text to raster formats via Agg Matplotlib relies on `FreeType
+<https://www.freetype.org/>`_.  Because the exactly rendering of the glyphs
```

Comment:
```suggestion
To output text to raster formats via Agg, Matplotlib relies on `FreeType
<https://www.freetype.org/>`_.  Because the exact rendering of the glyphs
```